test/e2e/cluster_staged_updaterun_test.go

```diff
@@ -1668,6 +1668,7 @@ func createClusterStagedUpdateRunSucceed(updateRunName, crpName, resourceSnapsho
 			PlacementName:            crpName,
 			ResourceSnapshotIndex:    resourceSnapshotIndex,
 			StagedUpdateStrategyName: strategyName,
+			State:                    placementv1beta1.StateStarted,
```

Comment:
shouldn't the default be "stateNotStarted"?  It's the default so we don't need to add it?

Reply:
Needed to add omitempty, fixex